reactor-netty-core/src/main/java/reactor/netty/channel/FluxReceive.java

```diff
@@ -268,12 +271,12 @@ final void drainReceiver() {
 				try {
 					if (logLeakDetection.isDebugEnabled()) {
 						if (v instanceof ByteBuf) {
-							((ByteBuf) v).touch(format(channel, "Receiver " + a.getClass().getName() +
-									" will handle the message from this point"));
+							((ByteBuf) v).touch(format(channel, RECEIVER_MESSAGE + a.getClass().getName() +
```

Comment:
In production where DEBUG is not enabled, these strings will not be created (guarded by `logLeakDetection.isDebugEnabled()`). If we convert these strings to a constants, then these constants will be created always regardless whether it is needed or not. Wdyt?